docs/getting-started/getting-started.mdx

```diff
@@ -1,78 +1,100 @@
+---
+title: Getting Started
+---
 
 ### Installation
+
 ```bash
 pip install tach
 ```
+
 ### Setup
+
 Tach allows you to configure where you want to place module boundaries in your project.
 
 You can do this interactively - run:
+
 ```bash
  tach mod
 # Up/Down: Navigate  Enter: Mark/unmark module  Right: Expand  Left: Collapse  Ctrl + Up: Jump to parent
 # Ctrl + s: Exit and save  Ctrl + c: Exit without saving  Ctrl + a: Mark/unmark all
 ```
+
 Mark each module boundary with 'Enter'. You can mark all of your top-level Python source packages, or just a few which you want to isolate.
 
 If your Python code lives below your project root, mark your Python [source root](https://gauge-sh.github.io/tach/configuration#source-root) using the 's' key.
 
 This will create the config file for your project, `tach.yml`.
 
 Once you've marked all the modules you want to enforce dependencies between, run:
+
 ```bash
 tach sync
 ```
+
 Dependencies that exist between each module you've marked will be written to `tach.yml`.
 
-Check out what Tach has found! 
+Check out what Tach has found!
+
 ```
 cat tach.yml
 ```
 
 Note: Your 'project root' directory (where `tach.yml` is) will be treated as a module boundary, and can show up as `<root>`.
 
 ### Enforcement
+
 Tach comes with a cli command to enforce the boundaries that you just set up! From the root of your Python project, run:
+
 ```bash
 tach check
 ```
+
 You will see:
+
 ```bash
 ✅ All module dependencies validated!
 ```
 
 You can validate that Tach is working by either:
+
 1. Commenting out an item in a `depends_on` key in `tach.yml`
-2. By adding an import between modules that didn't previously import from each other. 
+2. By adding an import between modules that didn't previously import from each other.
 
 Give both a try and run `tach check` again. This will generate an error:
+
 ```bash
-❌ tach/check.py[L8]: Cannot import 'tach.filesystem'. Tag 'tach' cannot depend on 'tach.filesystem'. 
+❌ tach/check.py[L8]: Cannot import 'tach.filesystem'. Tag 'tach' cannot depend on 'tach.filesystem'.
 ```
 
 Each error indicates an import which violates your dependencies. If your terminal supports hyperlinks, click on the file path to go directly to the error.
 
-`tach check` will also raise an error code. It can be easily integrated with CI/CD, [Pre-commit hooks](https://gauge-sh.github.io/tach/usage/#tach-install), and [VS Code](https://marketplace.visualstudio.com/items?itemName=Gauge.tach), and more!  
+`tach check` will also raise an error code. It can be easily integrated with CI/CD, [Pre-commit hooks](https://gauge-sh.github.io/tach/usage/#tach-install), and [VS Code](https://marketplace.visualstudio.com/items?itemName=Gauge.tach), and more!
```

Comment:
```suggestion
`tach check` will also return an error code. It can be easily integrated with CI/CD, [Pre-commit hooks](https://gauge-sh.github.io/tach/usage/#tach-install), and [VS Code](https://marketplace.visualstudio.com/items?itemName=Gauge.tach), and more!
```

nit: this wasn't from your changes but this tweak would be nice